qiskit/dagcircuit/_dagcircuit.py

```diff
@@ -1145,6 +1145,42 @@ def remove_nondescendants_of(self, node):
             if nd["type"] == "op":
                 self._remove_op_node(n)
 
+    def remove_edge(self, node1, node2, wire=None):
```

Comment:
It doesn't look like `wire` is optional here. (We'll always raise if `wire` is `None`). In that case, we should make it a required argument and update the docstring.

Reply:
I have updated the code so that wire is optional - it removes a random edge between the two given nodes.